### PR TITLE
Added 2 methods for number formatting.

### PR DESCRIPTION
**Pull request by kikoruiz**
Methods added:
* __`n`__: Formats a number regarding a given culture (e.g: for `es-ES`, `1000` will be __`1.000`__).
* __`c`__: Formats a number regarding a given culture and currency type (e.g: for `es-ES` and `EUR`, `1000` will be __`1.000 €`__).

It solves issue https://github.com/SUI-Components/rosetta/issues/21.

@SUI-Components/developers please review!

__NOTE:__ We should also think of adding some kind of polyfill for Node.js (server side rendering), because it only supports English locale by default: http://formatjs.io/guides/runtime-environments/#server

### CODE REVIEW

File: .sass-lint.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# sass-lint config generated by make-sass-lint-config v0.1.1
```
Comment (kikoruiz) [inline .sass-lint.yml]: Added by `frontend-precommit-rules` package.

**Comment by carlosvillu** [comment]
👍

**Comment by ruben-martin-lozano** [comment]
👍

**Comment by kikoruiz** [comment]
Updated README.md with the new methods.

**Pull request merged**